TCMSApp/src/client/app/defects/add-defect.controller.spec.js

```diff
@@ -0,0 +1,54 @@
+/* jshint -W117, -W030 */
+describe('Tests Add-Defects Open Model', function() {
```

Comment:
Tests Add-Defects Open Model check succesful creation of controller AddDefectController and test defect - of controller Defect. Two controllers, two tests. Maybe Tests Add-Defects Open Model is not correct name of test - Test AddDefectController Open Model Branch will be better